### PR TITLE
"Ansible for Networking" Lesson and Image

### PR DESCRIPTION
**Pull request by Mierdin**
Note - this implements part of #97 but there is room for more that this lesson might not cover, like specifics of Junos modules, better suited for a dedicated lesson. Should consider closing that issue when this gets merged **but** opening up new issues for those specific topics

# TODO

- [x] Change all paths (like images) that are pointing to the fork back to this repo

### CODE REVIEW

**Comment by Mierdin** [comment]
Things seem to be working pretty well but I did get this error at some point:

```
[antidote@ansible stage1]$ ansible-playbook -i devices playbook.yml

PLAY [Ansible test within NRE Labs lesson] **********************************************************

TASK [configure the login banner] *******************************************************************
fatal: [vqfx1]: FAILED! => {"ansible_facts": {"discovered_interpreter_python": "/usr/libexec/platform-python"}, "changed": false, "module_stderr": "", "module_stdout": "\n{\"msg\": \"name 'known_hosts_lookup' is not defined\", \"failed\": true, \"invocation\": {\"module_args\": {\"banner\": \"login\", \"port\": 22, \"text\": \"this is my login banner\\nthat contains a multiline\\nstring\\n\", \"state\": \"present\", \"provider\": {\"host\": null, \"port\": 22, \"username\": null, \"password\": null, \"ssh_keyfile\": null, \"timeout\": null, \"transport\": \"netconf\"}, \"active\": true, \"host\": null, \"username\": null, \"password\": null, \"ssh_keyfile\": null, \"timeout\": null, \"transport\": null}}, \"deprecations\": [{\"msg\": \"Param 'port' is deprecated. See the module docs for more information\", \"version\": 2.9}]}\n\n{\"msg\": \"name 'known_hosts_lookup' is not defined\", \"failed\": true, \"invocation\": {\"module_args\": {\"banner\": \"login\", \"port\": 22, \"text\": \"this is my login banner\\nthat contains a multiline\\nstring\\n\", \"state\": \"present\", \"provider\": {\"host\": null, \"port\": 22, \"username\": null, \"password\": null, \"ssh_keyfile\": null, \"timeout\": null, \"transport\": \"netconf\"}, \"active\": true, \"host\": null, \"username\": null, \"password\": null, \"ssh_keyfile\": null, \"timeout\": null, \"transport\": null}}, \"deprecations\": [{\"msg\": \"Param 'port' is deprecated. See the module docs for more information\", \"version\": 2.9}]}\n", "msg": "MODULE FAILURE\nSee stdout/stderr for the exact error", "rc": 1}

PLAY RECAP ******************************************************************************************
vqfx1                      : ok=0    changed=0    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0   
```

Unfortunately I can't seem to replicate it. When I re-ran the playbook, it ran normally.

**Comment by Mierdin** [comment]
Nevermind, I wasn't actually using the provided `ansible.cfg` file in the correct location. Moving this fixed the problem, works great now.

File: .gitignore [hunk 1]
```diff
@@ -45,3 +45,5 @@ __pycache__
 
 *.img
 *.tgz
+
+!lessons/tools/lesson-41-ansible-network/stage1/hosts
```
Comment (Mierdin) [inline .gitignore]: What's this for?

**Comment by IPvSean** [comment]
That’s the inventory file, for whatever reason the Ansible docs and our
training always used the file name hosts, so I wanted it to match so
students won’t get confused

You had it git ignored so I whitelisted just the single one I needed

**Comment by Mierdin** [comment]
Okay. I **desperately** need to update .gitignore in this repo, it's got a whole bunch of stuff in there that is a remnant of what used to be in this repo. Do what you need to do for now, but I'll likely come in after and clean all that crap up anyways

**Comment by Mierdin** [comment]
@IPvSean I think we're good to merge now. I took the liberty of adding a lesson diagram - mostly useful for the last two stages, but still good to have lesson-wide anyways, in case the learner wants to play around.

I'll make sure this is reflected on the test site immediately after I merge, to ensure everything still works, especially the pictures throughout the lesson, etc. If there's anything you wish to change prior to release, don't hesitate to let me know - we are still a few days out from that. 

Thanks so much for your quick work on this - I think this is going to be one of our signature lessons. I know it's been one of our most-requested.

**Pull request merged**